Sources/VGSCollectSDK/Core/Collector/VGSCollect.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  VGSForm.swift
+//  VGSCollectSDK
+//
+//  Created by Vitalii Obertynskyi on 8/26/19.
+//  Copyright © 2019 Vitalii Obertynskyi. All rights reserved.
+//
+
+import Foundation
+#if canImport(UIKit)
+import UIKit
+#endif
+//import Alamofire
```

Comment:
It should be removed? Or next time?

Reply:
We can remove everything related to Alamofire in next release.